website/src/.events.html

```diff
@@ -0,0 +1,81 @@
+<div class='alertstream'>
+  <h2>Events</h2>
+  <noscript>To see upcoming events please turn on javascript.</noscript>
+  <!-- Place each event on in it's own div -->
+  
+  <div class='alert'>
```

Comment:
@Shivank98 can we use this file to add new events that we'll be holding?

- Servs as a reminder list for all places where we have to do announcements.
- Quick links to all relevant announcements / call links
- People can know which events are live.

Reply:
Great Idea. is there some way people can save the event to their calendar?

Reply:
Aaah! I did not think of that!

I'll add google calendar integration. Should be good enough right?

Reply:
I also don't have an exact idea. if some integration is possible it is good. else there is also something like ics files, may be that can also help.

Reply:
From what I can tell, the simplest way to do this is to add a link to a public google calendar event. I'll do that in a later PR if someone else does not do it first?

Reply:
rest looks good. but that link would be of the complete calendar right. I think that is way to go. earlier I was thinking if something like user clicks the link and it automatically gets added to his calendar.

Reply:
No no. I meant a calendar event. Kind of like "invite to event".

I'll read a little more tomorrow about this.

Reply:
Ok. merging this.